<ui version="4.0" >
 <class>SMPPolyFacWidget</class>
 <widget class="QWidget" name="SMPPolyFacWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QGridLayout" name="gridLayout_2" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Roots algorithm:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Determinant algorithm:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QComboBox" name="rootsCombo" >
       <item>
        <property name="text" >
         <string>MÃ¼ller</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Bauhuber</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>LAPACK</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QComboBox" name="determinantCombo" >
       <item>
        <property name="text" >
         <string>Laplace</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Faddeev</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>444</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="1" >
      <widget class="QPushButton" name="computeButton" >
       <property name="text" >
        <string>Perform coefficient precomputation</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QProgressBar" name="progressBar" >
       <property name="maximum" >
        <number>1</number>
       </property>
       <property name="value" >
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
